person.rb

```diff
@@ -0,0 +1,17 @@
+class Person
+  attr_accessor :id, name, age
+
+  def can_use_services?
+    return unless age == 18 && parent_permission == true
+
+    true
+  end
+
+  private
+
+  def of_age?
+    return false if age <= 18
+
+    true
+  end
```

Comment:
- [ ] To avoid errors kindly use the `@age` instance variable here instead of  just`age`

- [ ] **[OPTIONAL]** This method can be improved and made concise:

  ```
  def of_age?
    @age >= 18
  end
  ```